### PR TITLE
Improve histogram making code - Add code to produce "3-panels plots"

### PR DESCRIPTION
**Pull request by fleble**
## Improve histogram making code
In directory `makeHistograms`
* Define variables to histogram using a set of filters and defines written in a json file. This is more concise and different sets of variables can be histogramed without touching the code. In particular, only a few variables that need to be added to the ROOT files can be defined/computed.
* Add argument parser
* Use ROOT file opening mode "UPDATE" to add more histograms to an already existing ROOT file
* Add comments in json files

## Add code to produce "3-panels plots"
In directory  `plottingTools/threePanelsPlots`
Make plots of a variable from its histograms for different processes with 3 panels:
* Top: histogram
* Middle: significance
* Bottom: efficiencies

For now:
* All samples, samples to stack, signal and background definition, labels for legend and colors are all defined at the top of `makePlots.py`.
* Histograms can be normalized to 1 by setting `NORMALIZE_TO_1 = True` at the top of `makePlots.py`. This affects significance.

### CODE REVIEW

File: plottingTools/common/histograms.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
```
Comment (jniedzie) [inline plottingTools/common/histograms.py:1]: The name of this file is not very descriptive - I haven't found any histograms inside ;)

File: makeHistograms/run.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+variables=variables.json
```
Comment (jniedzie) [inline makeHistograms/run.sh:1]: Name of the file could be more descriptive - "run" what?

Comment (fleble) [reply makeHistograms/run.sh:1]: Would you prefer `runMakePlots.sh`?

**Review by jniedzie — APPROVED** [review]
Yes, new names look much better!

**Pull request merged**